akka-stream/src/main/scala/akka/stream/io/SslTls.scala

```diff
@@ -138,14 +141,14 @@ object SslTls {
  * unwrapping [[SendBytes]].
  */
 object SslTlsPlacebo {
-  val forScala = scaladsl.BidiFlow() { implicit b ⇒
+  val forScala: SslTls.ScalaFlow = scaladsl.BidiFlow() { implicit b ⇒
```

Comment:
This made the type more generic than it used to be leading to the test compilation failure: 

before its type was

```scala
scaladsl.BidiFlow[SslTlsOutbound, ByteString, ByteString, SessionBytes, Unit]
```

now it's

```scala
scaladsl.BidiFlow[SslTlsOutbound, ByteString, ByteString, SslTlsInbound, Unit]
```